### PR TITLE
feat: dev-wallet

### PR DESCRIPTION
**Pull request by javadkh2**
We're in the process of creating a wallet that specifically caters to Kadena's unique requirements. Initially, our focus has been on addressing the needs of dApp developers. However, we're also refining the user experience for a broader audience.

Currently, our code is based on a boilerplate from the POC we did, which serves as our starting point. It's important to note that we anticipate making significant changes to adapt and enhance this codebase.

The key objectives for our "dev-wallet" development include:

1. **Multi-Chain Architecture**: We aim to offer a high-level approach for managing multi-chain architecture efficiently.

2. **Account Discovery**: We're working on an account discovery method to simplify user interactions.

3. **Multiple Wallet Creation**: Our goal is to empower users to create multiple wallets, ensuring flexibility and security.

4. **Network Support**: We're committed to providing support for development networks and custom networks, enabling users to tailor their experience.

5. **Key Pair Import**: Users will have the ability to import key pairs from other wallets, facilitating a seamless transition.

6. **Chainweaver Features**: Our wallet will also incorporate features related to accounts that align with Chainweaver's functionality.

### CODE REVIEW

File: packages/apps/dev-wallet/desktop/src-tauri/icons/128x128.png
Comment (alber70g) [inline packages/apps/dev-wallet/desktop/src-tauri/icons/128x128.png:1]: Can we remove the icons and pngs, and squash this PR so they don't pollute the repo (in size)

**Review by alber70g — APPROVED** [review]

**Review by alber70g — APPROVED** [review]

File: packages/apps/dev-wallet/desktop/src-tauri/icons/128x128.png
Comment (javadkh2) [reply packages/apps/dev-wallet/desktop/src-tauri/icons/128x128.png:1]: so where should I put them? we need the icons?

File: packages/libs/react-ui/src/components/Form/Select/Select.test.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ describe('Select', () => {
     await userEvent.click(selectContainer);
     await userEvent.click(screen.getByRole('option', { name: 'Option 2' }));
     expect(handleChange).toHaveBeenCalledTimes(1);
-  });
+  }, 20000);
```
Comment (javadkh2) [inline packages/libs/react-ui/src/components/Form/Select/Select.test.tsx:62]: it just throws timeout on CI without increasing the timeout

Comment (Ghislain89) [reply packages/libs/react-ui/src/components/Form/Select/Select.test.tsx:62]: This is fine, the test is also slow locally. We might have to think about increasing the size of our GitHub runners

**Review by Ghislain89 — APPROVED** [review]

**Review by KristinaSpasevska — APPROVED** [review]

**Review by realdreamer — APPROVED** [review]

**Review by salamaashoush — APPROVED** [review]

**Review by sstraatemans — APPROVED** [review]

**Review by nil-amrutlal-dept — APPROVED** [review]

**Pull request merged**